interbtc-spec/docs/source/spec/nomination.rst

```diff
@@ -5,84 +5,61 @@ Vault Nomination
 
 Overview
 ~~~~~~~~
-Vault Nomination is a feature aimed at increasing `interbtc` issuance capacity which introduces two actors: Nominators and Operators.
-Vaults who opt in to this feature take on the additional role of nomination Operators. 
-A Nominator is anyone who locks their free collateral so that Operators they trust can issue `interbtc` backed by the nominated collateral.
-Nominators are rewarded a fraction of the fees generated by their collateral, while the rest of the fees is given to the Operator.
-Operators are assumed to be trusted by their nominators not to steal Bitcoin backed by nominated collateral.
-
+Nomination is a feature aimed at increasing `interbtc` issuance capacity by allowing Nominators to back a particular Vault.
+They lock their free collateral so that trusted Vaults can issue `interbtc` backed by the nominated collateral.
+Nominators are rewarded a fraction of the fees generated by the Vault, while the remaining fees are given to the Vault.
+Vaults are assumed to be trusted by their Nominators not to steal Bitcoin backed by nominated collateral.
 
 Step-by-step
 ------------
 
-#. Vaults opt in to the nomination feature, becoming Operators.
-#. The maximum nomination an Operator can receive is bounded by their own locked collateral.
-#. Nominators select one or more Operators and lock their collateral balance onto the BTC Parachain.
+#. Vaults may opt in to Nomination, expanding the total possible issuance amount.
+#. The maximum amount that can be nominated is bounded by the Vault's locked collateral.
+#. Nominators select one or more Vaults and lock their collateral balance on the BTC Parachain.
 #. Nominators can go offline and their nominated collateral will generate rewards passively.
-#. Operator and Nominator collateral cannot be withdrawn directly. Rather, withdrawals are subject to an unbonding period.
-#. In case of Operator failure, Nominators are returned any left-over collateral (after victim users are reimbursed).
+#. Vaults and Nominators can withdraw their collateral at any point subject to the ``SecureCollateralThreshold``.
+#. Upon liquidation, Nominators are returned some collateral after remaining requests have been executed.
 
 
 Protocol
 ~~~~~~~~
 
-
 Security Assumptions and Considerations
 ---------------------------------------
 
 #. The operating Vault is trusted by its Nominators not to steal the interbtc issued with their collateral.
 #. There is no transitive trust. If a user trusts Vault A and Vault A trusts Vault B, the user does not trust Vault B.
 #. Nominators are mostly-offline agents, who are slow to respond to system changes.
 #. Vaults are always-online agents, who can promptly react to system updates.
-#. A Nominator may expose the Vault and the other Nominators to additional economic risk by withdrawing nominated collateral during an exchange rate spike. Similarly, the Vault may expose its Nominators to additional economic risk by withdrawing excess collateral.
```

Comment:
Good point, I think it is a concern in theory but maybe not practice - if the Vault is expected to be online it should be able to compensate appropriately. That being said, me and @savudani8 had discussed adding some buffer over the 150% collateralization.

Reply:
Think about the following case:
- DOT price drops relative to BTC & Nominators panic and exit nomination as fast as they can
- Vault is left with 150% collateralization just from nominators alone leaving the system
- Exchange rate drop is only reflected about 1 hour after the actual drop is happening (if we adopt Maker style oracles)
- Now the vault is left at 150% minus the exchange rate drop which might bring him under premium redeem or in the worst case below the liquidation threshold

Please leave the text in since I think it is a practical risk